docs/history.md

```diff
@@ -34,4 +34,12 @@ move the repository, so we simply forked it. We still welcome the HackMD team
 as part of our community, especially since a large portion of this code base
 originated with them.
 
-*For the debate that lead to this step, please refer to the [governance debate](https://github.com/hackmdio/hackmd/issues/1170) and [the announcement of the new repository](https://github.com/codimd/server/issues/10).*
+*For the debate that lead to this step, please refer to the [governance debate](https://github.com/hackmdio/hackmd/issues/1170) and [the announcement of the new repository](https://github.com/hedgedoc/hedgedoc/issues/10).*
+
+## CodiMD became HedgeDoc
+
+Many people opened issues in the codimd server repository for bugs and features, which they saw at the HackMD codimd.
```

Comment:
Maybe this sentence should start with something like "With two projects, both actively maintained, one by the HackMD team, and this hardfork by the community, confusion about the name and feature set became visible in form of issue reports about HackMD's version this repository, which as originally called `codimd/server`."